diff --git a/streamly.cabal b/streamly.cabal
index e199a26..fab4dcb 100644
--- a/streamly.cabal
+++ b/streamly.cabal
@@ -603,7 +603,7 @@ library
                      , atomic-primops    >= 0.8   && < 0.9
                      , lockfree-queue    >= 0.2.4 && < 0.3

-                     , unicode-data      >= 0.1   && < 0.4
+                     , unicode-data      >= 0.1   && < 0.5

                     -- Network
                      , network           >= 2.6   && < 3.2
