diff --git a/src/EulerHS/Framework/Flow/Interpreter.hs b/src/EulerHS/Framework/Flow/Interpreter.hs
index 36a537bc..3b9473ce 100644
--- a/src/EulerHS/Framework/Flow/Interpreter.hs
+++ b/src/EulerHS/Framework/Flow/Interpreter.hs
@@ -1,4 +1,3 @@
-{-# OPTIONS_GHC -Werror -Wno-redundant-constraints #-}

 module EulerHS.Framework.Flow.Interpreter
   ( -- * Flow Interpreter
diff --git a/src/EulerHS/Framework/Flow/Language.hs b/src/EulerHS/Framework/Flow/Language.hs
index fff32353..35814360 100644
--- a/src/EulerHS/Framework/Flow/Language.hs
+++ b/src/EulerHS/Framework/Flow/Language.hs
@@ -1,4 +1,3 @@
-{-# OPTIONS_GHC -Werror #-}
 {-# LANGUAGE AllowAmbiguousTypes        #-}
 {-# LANGUAGE DerivingStrategies         #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
